examples/hotrod/README.md

```diff
@@ -65,6 +65,15 @@ docker run \
 
 Then open http://127.0.0.1:8080
 
+### Run HotROD from kubernetes
+
+```bash
+kustomize build ../kubernetes | kubectl apply -f -
```

Comment:
examples/ folder is not limited to hotrod, couldn't this be under examples/hotrod/kubernetes ?

Reply:
Moved in https://github.com/jaegertracing/jaeger/pull/3645/commits/4e7ce011354c2dbbfd6eb28f077d32e92423d88c